spec/models/comment_spec.rb

```diff
@@ -167,6 +167,92 @@
     end
   end
 
+  context "when user has disabled guest replies" do
+    let(:no_reply_guy) do
+      user = create(:user)
+      user.preference.update!(guest_replies_off: true)
+      user
+    end
+
+    let(:guest_reply) do
+      Comment.new(commentable: comment,
+                  pseud: nil,
+                  name: "unwelcome guest",
+                  email: Faker::Internet.email,
+                  comment_content: "I'm a vampire.")
+    end
+
+    let(:user_reply) do
+      Comment.new(commentable: comment,
+                  pseud: create(:user).default_pseud,
+                  comment_content: "Hmm.")
+    end
+
+    shared_examples "creating guest reply is allowed" do
+      describe "save" do
+        it "allows guest replies" do
+          expect(guest_reply.save).to be_truthy
+          expect(guest_reply.errors.full_messages).to be_blank
+        end
+        it "allows user replies" do
```

Comment:
I think we need a blank line between `end` and the next example -- same on line 210.